Dockerfile

```diff
@@ -1,44 +1,55 @@
-ARG dialect
+ARG dialect=sqlite
+
+FROM python:slim as poetry
+
+ARG dialect=sqlite
```

Comment:
Why this duplicates?

Reply:
Because of docker stupid useless ARG visibility levels